src/validation/support/contactUs.ts

```diff
@@ -0,0 +1,8 @@
+import { object, string } from 'yup'
+
+export const contactUsValidationSchema = object({
+  fullName: string().required('Full name is required'),
+  email: string().email().required('Email is required'),
```

Comment:
this will not support correct validation of email, let's right manually regexp for it